tests/lib/snaps/test-snapd-remodel-pc-24/meta/gadget.yaml

```diff
@@ -0,0 +1,57 @@
+volumes:
+  pc:
+    # bootloader configuration is shipped and managed by snapd
+    bootloader: grub
+    structure:
+      - name: mbr
+        type: mbr
+        size: 440
+        update:
+          edition: 1
+        content:
+          - image: pc-boot.img
+      - name: BIOS Boot
+        type: DA,21686148-6449-6E6F-744E-656564454649
+        size: 1M
+        offset: 1M
+        offset-write: mbr+92
+        update:
+          edition: 2
+        content:
+          - image: pc-core.img
```

Comment:
This partition is just empty in UC24. Remove the content.

Reply:
Where did you get this file from?

Reply:
copied from the uc22 one

Reply:
content should be removed?

Reply:
You could have kept the "update" entry. But it is fine without.